npc/re/instances/octopus_cave.txt

```diff
@@ -43,63 +43,44 @@ mal_dun01,151,235,5	script	Starfish	4_ASTER,{
 
 	if (!.@party_id) {
 		mes("[Starfish]");
-		mes("You alone is powerless, hehe!");
-		mes("Better get someone to help you out.");
-		mes("Make a party, and come back later.");
+		mes("You alone is powerless, hehe! Better get someone to help you out. Make a party, and come back later.");
```

Comment:
I think there's a lot of confusion about the whole `\r` matter, for which I apologize.

Many Aegis scripts introduce hard wraps, in the places where they want the lines to be split. This, in private servers, is desirable for some, and undesirable for others (especially when translating, or when using a different font in the client). For example:

```
mes("[Parmy Gianino]");
mes("Well then, would");
mes("you like to apply");
mes("for registration?");
```

Client output:

> [Parmy Gianino]
> Well then, would
> you like to apply
> for registration?

This produces 4 separate lines in the client output, which is fine in the original language and with the original font. If you use a larger font, those lines might get additionally split in other places, or if you translate it, the translated sentence might be longer or shorter. Additionally, those sentences ("you like to apply", "for registration?") are quite meaningless to translate, without enough context. That's really undesirable.

For that reason, we introduced the `\r`, which is a special character that we can decide whether to strip out from the script when parsing it (we have a setting in the emulator), so that, those who want to preserve the exact line breaks as in official servers, don't have to do anything; those who want to let the client format it properly regardless of language and font, can toggle the setting and strip the `\r`.

```
mes("[Parmy Gianino]");
mes("Well then, would\r"
    "you like to apply\r"
    "for registration?");
```

Client output (default):

> [Parmy Gianino]
> Well then, would
> you like to apply
> for registration?

Client output (optional, for best HULD or custom font output):

> [Parmy Gianino]
> Well then, would you like to apply for registration?

This can produce either 4 separate lines, or just two, and always produces a meaningful string to translate in the generated_translations.pot ("Well then, would\ryou like to apply\rfor registration?"), which contains enough context so that people are able to translate it properly, and even allows for languages that require more or less than 3 lines.

Now, this only makes sense if the sentence is the same. If you have separate sentences, there's no point in using \r to join them. They're separate sentences, and each of them works on its own. Additionally, the line break to separate two different sentences, is likely desirable for everyone, regardless of font and language. That's why I'd keep them in separate `mes` commands, instead of merging them.

Of course, when you don't want to introduce a line break between two sentences, then it's fine to merge them into just one `mes`.

I hope this makes sense, otherwise please ask me for clarification. (I'll copy this content into wiki page later on, if it's of any help)